tests/rptest/services/redpanda.py

```diff
@@ -2601,7 +2601,7 @@ def raise_on_bad_logs(self, allow_list=None):
         if allow_list is None:
             allow_list = DEFAULT_LOG_ALLOW_LIST
         else:
-            combined_allow_list = DEFAULT_LOG_ALLOW_LIST
+            combined_allow_list = DEFAULT_LOG_ALLOW_LIST.copy()
```

Comment:
The reason being that

> combined_allow_list.append(a)

is actually appending to what is intended to be a constant (the default allow list variable) ?

Reply:
Yes:

```python
>>> FOO = [1, 2, 3]
>>> a = FOO
>>> a.append(4)
>>> FOO
[1, 2, 3, 4]
```

Reply:
silly python